server/src/main/java/com/aws/greengrass/cli/CLIEventStreamAgent.java

```diff
@@ -435,6 +444,7 @@ protected void onStreamClosed() {
         @SuppressWarnings({"PMD.PreserveStackTrace", "PMD.AvoidCatchingGenericException"})
         public CreateLocalDeploymentResponse handleRequest(CreateLocalDeploymentRequest request) {
             return translateExceptions(() -> {
+                logger.atDebug().log("Creating new local deployment");
```

Comment:
is this something you forgot to remove?

Reply:
Oh yeah, my bad. Did not notice this got here.